libraries/AP_Scripting/lua_generated_bindings.cpp

```diff
@@ -483,6 +483,7 @@ static int GCS_send_text(lua_State *L) {
     const char * data_3 = luaL_checkstring(L, 3);
     ud->send_text(
             data_2,
+            "%s",
```

Comment:
You can't do this. It's a generated file. Stuff got merged today that should let us cope with this however. I will try and poke it in the next couple of days.

Reply:
Sooo.... can we fix the generated code now and you can fix the generator later? :-P

Reply:
No. I'm pretty sure the generator can emit this now for you (or needs a tiny modification), but I need to test that when I'm more awake. But just editing the file is pointless because it's going to get hosed repeatedly.

Reply:
#11877

Reply:
#11877 has been merged.